src/resources/subscription-manager-nose-tests.sh

```diff
@@ -16,8 +16,8 @@
 echo "sha1:" "${sha1}"
 
 # Decide which package manager to use
-which dnf || EXITCODE=$?
-if [ $EXITCODE -eq 0 ]; then
+which dnf
```

Comment:
This will fail on systems which don't have dnf and in bash shells that have 'set -e' the remainder of the script won't run (which it seems like jenkins does for us).

Beyond that my original attempt to switch between dnf and yum was unnecessary b/c apparently the fedora folks have made the choice to keep yum around, so yum will exist in all distros we intend to run our tests on.

Reply:
BTW: I  did some experiments...

When `set -e` is not used:

```bash
[jhnidek@localhost ~]$ true || RESULT=$?
[jhnidek@localhost ~]$ echo $RESULT

[jhnidek@localhost ~]$ false || RESULT=$?
[jhnidek@localhost ~]$ echo $RESULT
1
```

When `set -e` is used:
```bash
[jhnidek@localhost ~]$ set -e
[jhnidek@localhost ~]$ true || RESULT=$?
[jhnidek@localhost ~]$ echo $RESULT
1
[jhnidek@localhost ~]$ false || RESULT=$?
[jhnidek@localhost ~]$ echo $RESULT
1
```